sync_coda_to_engagement_db.py

```diff
@@ -12,6 +12,8 @@
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description="Syncs data from Coda to an engagement database")
 
+    parser.add_argument("--incremental-cache-path",
```

Comment:
What's the benefit of making this optional?

Reply:
Primarily in development - it's easier to think about caches only when you need them rather than constantly needing to create and specify folders for them, remember what's in them, and manually delete them before every run.